tests/examples/test_examples.py

```diff
@@ -69,7 +69,7 @@ def report_free_disk_space(capsys):
     yield
 
     with capsys.disabled():
-        print(" | Free disk space: {:.1f} GiB".format(get_free_disk_space()), end="")
```

Comment:
This prints out available disk space after running each example. Let's keep this line.

![image](https://user-images.githubusercontent.com/17039389/161237476-bdf39dec-0041-489f-9acf-2d09be89b589.png)

Reply:
I have tried, but when i put the `# pylint: disable-next=print-function` comment, it becomes with the following error: 
```
************* Module test_examples
tests/examples/test_examples.py:71:0: E0012: Bad option value 'print-function' (bad-option-value)
```

I have also tried with `# pylint: disable=print-function` and `# pylint: disable=print-statement`

Any other possible solution?

Reply:
Strange, have you tried this?

```
# pylint: disable=print-function
print(...)
```

Reply:
I just tried:
´´´
with capsys.disabled():
        # pylint: disable=print-function
        print(" | Free disk space: {:.1f} GiB".format(get_free_disk_space()), end="")
´´´

And I get this also:
´´´
Running pylint...
************* Module test_examples
tests/examples/test_examples.py:72:0: E0012: Bad option value 'print-function' (bad-option-value)
´´´

I had this pylint:
```

Reply:
```
    with capsys.disabled():
        # pylint: disable=print-function
        print(" | Free disk space: {:.1f} GiB".format(get_free_disk_space()), end="")
```

This worked in my environment. I also use pylint 2.11.1.

Reply:
`# pylint: disable-next=print-function` also worked.

Reply:
@harupy I'm getting this pylint failure on a merge from master on https://github.com/mlflow/mlflow/pull/5553 with this pylint comment in mlflow.utils.logging_utils.py:90. Removing the pylint disable check made pylint pass. Seems as though the trigger didn't happen so the disable check failed on parsing within the eprint() function.

Reply:
> Removing the pylint disable check made pylint pass.

This indicates that the print-function rule is not enabled.

Reply:
Done!